### PR TITLE
Added support for Sylius 1.11, dropped PHP7.4, Symfony version upgraded to 5.4

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.0",
         "knplabs/knp-snappy-bundle": "^1.7",
         "myclabs/php-enum": "^1.7",
         "sylius/resource-bundle": "^1.6",
-        "sylius/sylius": "^1.10",
-        "symfony/messenger": "^4.4 || ^5.3"
+        "sylius/sylius": "^1.10 | ^v1.11.0-alpha",
```
Comment (lchrusciel) [inline composer.json:26]:
```suggestion
        "sylius/sylius": "^1.10 || ^v1.11.0-alpha",
```

File: tests/Application/.env [hunk 2]
```diff
@@ -31,3 +31,10 @@ JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
 JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
 JWT_PASSPHRASE=66d45daf91b2ed1031e62d81c81dba2e
 ###< lexik/jwt-authentication-bundle ###
+
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+MESSENGER_TRANSPORT_DSN=doctrine://default
+# MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+###< symfony/messenger ###
```
Comment (lchrusciel) [inline tests/Application/.env:40]:
```suggestion
###< symfony/messenger ###

```

File: tests/Application/.env.test [hunk 3]
```diff
@@ -5,3 +5,7 @@ KERNEL_CLASS='Tests\Sylius\RefundPlugin\Application\Kernel'
 ###> knplabs/knp-snappy-bundle ###
 WKHTMLTOPDF_PATH="%kernel.project_dir%/etc/wkhtmltopdf"
 ###< knplabs/knp-snappy-bundle ###
+
+###> symfony/messenger ###
+MESSENGER_TRANSPORT_DSN=sync://
+###< symfony/messenger ###
```
Comment (lchrusciel) [inline tests/Application/.env.test:11]:
```suggestion
###< symfony/messenger ###

```

**Review by AdamKasp — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Closing in favour of #346

**Pull request closed without merging**